clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesApiAdaptor.groovy

```diff
@@ -51,6 +64,12 @@ class KubernetesApiAdaptor {
     this.config = config
     this.account = account
     this.client = new DefaultKubernetesClient(this.config)
+
+    this.newClient = io.kubernetes.client.util.Config.defaultClient()
```

Comment:
This will not respect with kubeconfig, context, user, or cluster will be chosen.